README.md

```diff
@@ -85,21 +85,19 @@ let res = dynamo_client
 aws_span.end(&res);
 ```
 
-Defining a custom aws target:
+Only the following AWS targets are fully supported at the moment:
 
-```rust
-struct S3Target {}
-impl IntoAttributes for S3Target {
-    fn service(&self) -> &'static str {
-        "s3"
-    }
-
-    fn into_attributes(self, _method: &'static str) -> Vec<KeyValue> {
-        vec![semcov::AWS_S3_BUCKET.string("my_bucket")]
-    }
-}
-```
+ * DynamoDB
+ * SNS
+ * Firehose
+
+But a generic `AwsOperation` could be used to instrument any other AWS SDK:
 
 ```rust
-let s3_span = AwsSpan::new(S3Target {}, "GetObject");
+let s3_span = AwsOperation::client(
+    "S3",
+    "GetObject",
+    vec![semcov::AWS_S3_BUCKET.string("my_bucket")],
+)
```

Comment:
And this is a new generic operation which could be used to instrument any AWS SDK.

Previously the same functionality was exposed through `IntoAttributes` trait, but it looks better this way.